src/device-uptime/utils/device_hourly_records.py

```diff
@@ -60,56 +46,27 @@ def get_sensor_readings(self):
                 battery_voltage=[]
             )
 
-        self.df['time'] = pd.to_datetime(self.df['time'])
-        time_indexed_data = self.df.set_index('time')
-
-        # change frequency to hours
-        hourly_data = time_indexed_data.resample('H').mean().round(2)
-        daily_data = hourly_data.resample('D').mean().dropna()
-        sensor_one_pm2_5 = daily_data['s1_pm2_5'].tolist()
-        sensor_two_pm2_5 = daily_data['s2_pm2_5'].tolist()
-        battery_voltage = daily_data['voltage'].tolist()
-        time = daily_data.index.tolist()
-
         return DeviceSensorReadings(
-            time=time,
-            sensor_one_pm2_5=sensor_one_pm2_5,
-            sensor_two_pm2_5=sensor_two_pm2_5,
-            battery_voltage=battery_voltage,
+            time=self.record.get("created_at"),
+            sensor_one_pm2_5=self.record.get("pm2_5"),
+            sensor_two_pm2_5=self.record.get("s2_om2_5"),
```

Comment:
please review this line, is this supposed to be s2_pm2_5? @ja-odur

Reply:
Nice catch @Baalmart

Reply:
@Baalmart This has been fixed